lib/nerves_hub_link.ex

```diff
@@ -1,4 +1,7 @@
 defmodule NervesHubLink do
+  alias __MODULE__.Supervisor, as: NHLSupervisor
```

Comment:
any reason to use `__MODULE__` here instead of just using the module's name? i don't have any serious opinions on it, but it's not something i see often.

Reply:
Nope. Just happen I guess